documents/write_acls.go

```diff
@@ -84,3 +86,121 @@ func newChangedField(p proofs.Property, leaf *proofs.LeafNode, old bool) changed
 	cf.new = v
 	return cf
 }
+
+// transitionRulesFor returns a copy all the transition rules for the account.
+func (cd *CoreDocument) transitionRulesFor(account identity.DID) (rules []coredocumentpb.TransitionRule) {
+	for _, rule := range cd.Document.TransitionRules {
+		for _, rk := range rule.Roles {
+			role, err := getRole(rk, cd.Document.Roles)
+			if err != nil {
+				continue
+			}
+
+			if _, ok := isAccountInRole(role, account); !ok {
```

Comment:
Following comment above: `is[DID|ID]InRole` fits better.

Reply:
@mikiquantum This is something we need to discuss as putting `is[DID|ID]InRole` just fixes the name to a DID, which is an implementation of centrifuge account IMO.